src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts

```diff
@@ -0,0 +1,39 @@
+import * as AdhEmbedModule from "../../../Embed/Module";
+import * as AdhPermissionsModule from "../../../Permissions/Module";
+import * as AdhResourceAreaModule from "../../../ResourceArea/Module";
+import * as AdhTopLevelStateModule from "../../../TopLevelState/Module";
+
+import * as AdhEuthWorkbenchModule from "../Workbench/Module";
+
+import * as AdhEmbed from "../../../Embed/Embed";
+import * as AdhResourceArea from "../../../ResourceArea/ResourceArea";
+
+import * as AdhEuthWorkbench from "../Workbench/Workbench";
+
+import RIPeuthProcess from "../../../../Resources_/adhocracy_euth/resources/idea_collection/IProcess";
```

Comment:
How can I document a convention that I don't know? Where does the 'I' stand for?